orchestrator.go

```diff
@@ -89,6 +90,9 @@ type Learnuplet struct {
 
 /*
  * The Init method is called when the Smart Contract orchestrator is instantiated by the blockchain network
+ * Note that chaincode upgrade also calls this function to reset
```

Comment:
The init method is like this, for test purposes, waiting for Mattthieu to be done with updating the tests